content/articles/2019-05-vroom-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,450 @@
+    theme_grey(base_size = 18) +
+    theme(legend.position = "bottom") +
+    coord_flip() +
+    labs(title = title, subtitle = subtitle, x = NULL, y = NULL, fill = NULL)
+}
+
+make_table <- function(data) {
+  times <- data %>%
+    group_by(package, op) %>%
+    filter(type == "real") %>%
+    tally(wt = time) %>%
+    spread(op, n) %>%
+    mutate(total = sum(read, print, head, tail, sample, filter, aggregate))
+
+  times %>%
+    arrange(desc(total)) %>%
+    mutate_if(is.numeric, pretty_sec) %>%
+    knitr::kable(digits = 2, align = "r", format = "markdown")
+}
+
+desc <- c("setup", "read", "print", "head", "tail", "sample", "filter", "aggregate")
+```
+
+<html>
+<style>
+h2 code {
+    font-size: 1em;
+}
+</style>
+</html>
+
+I'm excited to announce that [vroom 1.0.0](http://vroom.r-lib.org) is now
+available on CRAN!
+
+vroom reads rectangular data, such as comma separated
+(csv), tab separated (tsv) or fixed width files (fwf) into R.
+
+It performs the
+same function as packages like [readr](http://readr.r-lib.org),
+[data.table](http://r-datatable.com) or the `read.csv()`
+function, but for many datasets vroom can read them much, much
+faster (hence the name).
+
+The main reason vroom can be faster is because character data is read from the
+file lazily; you only pay for the data you use. This lazy access is done
+automatically, so no changes to your R data manipulation code are needed.
+
+vroom also provides efficient multi-threaded writing that is multiple times
+faster on most inputs than the readr writer.
+
+Install vroom with:
+
+```{r, eval = FALSE}
+install.packages("vroom")
+```
+
+The best way to get acquainted with the package is the [getting
+started](http://vroom.r-lib.org/articles/vroom.html) vignette.
+
+## Reading delimited files
+
+Compared to readr, the first difference you may note is you use only one
+function to read the files, `vroom()`. This is because `vroom()` guesses the
+delimiter of the file automatically (based on the first few lines). This works
+well most of the time, but may fail to guess properly in some cases. The
+`delim` argument is used to specify the delimiter of the file explicitly.
+
+```{r}
+library(vroom)
+
+data <- vroom("flights.tsv")
+```
+
+The summary message after reading also differs from readr. We hope this output
+gives a more informative indication if the types of your columns are being guessed
+properly. However you can still retrieve and print the full column specification
+with `spec()`.
+
+```{r}
+spec(data)
+```
+
+The message will be disabled if you supply a column specification to `col_types` when reading.
+
+```{r}
+s <- spec(data)
+
+data <- vroom("flights.tsv", col_types = s)
+```
+
+## Reading multiple files
+
+One feature new to vroom is built-in support for reading sets of files with the
+same columns into one table. Just pass the filenames to be read directly to
+vroom.
+
+```{r}
+# Split the flights data by carrier
```

Comment:
I'd hide this code, and instead say something like: "Imagine we have a directory containing ..."